### PR TITLE
:white_check_mark: 'fs' api

### PR DESCRIPTION
**Pull request by bpetetot**
#6

### CODE REVIEW

File: src/server/api/models/fs/fs.spec.js [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+    const result = await service.ls('/foo/bar')
+
+    // asserts
+    expect(result).toBeDefined()
+    expect(result.length).toBe(0)
+  })
+
+  it('should give fullname, name, ext and isDirectory metadata', async () => {
+    // mocks
+    fs.readdirAsync.mockImplementation(jest.fn(async () => ['baz.txt']))
```
Comment (fabienjuif) [inline src/server/api/models/fs/fs.spec.js]: You can use this it to test `isDirectory : false`

File: src/server/api/models/fs/fs.spec.js [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+
+    // asserts
+    expect(result).toBeDefined()
+    expect(result.length).toBe(1)
+    expect(result[0].fullname).toBe('baz.txt')
+    expect(result[0].name).toBe('baz')
+    expect(result[0].ext).toBe('.txt')
+    expect(result[0].isDirectory).toBe(true)
+  })
 })
```
Comment (fabienjuif) [inline src/server/api/models/fs/fs.spec.js]: One test is missing to me : that stats properties are cloned into the return value.

**Review by fabienjuif — CHANGES_REQUESTED** [review]
See comments

**Review by fabienjuif — APPROVED** [review]

**Comment by bpetetot** [comment]
@fabienjuif you can check it

**Pull request merged**